protocol/messenger_sync_settings.go

```diff
@@ -159,3 +159,24 @@ func (m *Messenger) startSyncSettingsLoop() {
 		}
 	}()
 }
+
+func (m *Messenger) startSettingsChangesLoop() {
+	channel := m.settings.SubscribeToChanges()
+	go func() {
+		for {
+			select {
+			case s := <-channel:
+				switch s.GetReactName() {
+				case settings.DisplayName.GetReactName():
+					m.selfContact.DisplayName = s.Value.(string)
+				case settings.PreferredName.GetReactName():
+					m.selfContact.EnsName = s.Value.(string)
+				case settings.Bio.GetReactName():
+					m.selfContact.Bio = s.Value.(string)
```

Comment:
Is there a chance that somewhere else we might modify those values? I'm asking in case we need a lock

Reply:
You mean `m.selfCotnact.EnsName = ...`? Then no, we only set those values here.
We also set it during initialisation, but that should be fine.